packages/state-transition/src/slot/upgradeStateToElectra.ts

```diff
@@ -93,7 +90,23 @@ export function upgradeStateToElectra(stateDeneb: CachedBeaconStateDeneb): Cache
   });
 
   for (const validatorIndex of preActivation) {
-    queueEntireBalanceAndResetValidator(stateElectraView as CachedBeaconStateElectra, validatorIndex);
+    const stateElectra = stateElectraView as CachedBeaconStateElectra;
+    const balance = stateElectra.balances.get(validatorIndex);
+    stateElectra.balances.set(validatorIndex, 0);
+
+    const validator = stateElectra.validators.get(validatorIndex);
+    validator.effectiveBalance = 0;
+    stateElectra.epochCtx.effectiveBalanceIncrementsSet(validatorIndex, 0);
+    validator.activationEligibilityEpoch = FAR_FUTURE_EPOCH;
+
+    const pendingDeposit = ssz.electra.PendingDeposit.toViewDU({
+      pubkey: validator.pubkey,
+      withdrawalCredentials: validator.withdrawalCredentials,
+      amount: balance,
+      signature: new Uint8Array(G2_POINT_AT_INFINITY),
```

Comment:
this seems like an unnecessary conversion

we should probably convert everything to Uint8Array here as well
https://github.com/ChainSafe/lodestar/blob/unstable/packages/state-transition/src/constants/constants.ts#L1